src/v0/destinations/braze/transform.js

```diff
@@ -166,6 +166,7 @@ function getUserAttributesObject(message, mappingJson, destination) {
   // reserved keys : already mapped through mappingJson
   const reservedKeys = [
```

Comment:
`reservedKeys` can be generated from `values` of `BrazeUserAttributesConfig.json` to always be in sync

Reply:
There is a difference, in reservedKeys we have RS keys while in mapping we have Braze -> RS with nested address properties too.

Reply:
For nested properties was seeing issues when referencing a common object for both

Reply:
For nested ones, you can get the prefix, and maintain a Set to keep distinct values. You can do this change later.

Reply:
Yes sure.